test/test_utils/sagemaker.py

```diff
@@ -322,7 +289,6 @@ def execute_local_tests(image, pytest_cache_params):
                     f"Image pull for {image} failed.\ndocker images output = {output}"
                 ) from e
         ec2_conn.run(f"tar -xzf {sm_tests_tar_name}")
-        kill_background_processes_and_run_apt_get_update(ec2_conn)
```

Comment:
Wanted to double check that we don't see any problems w/ moving this func?

Reply:
We're not removing it, as such. We've just changed the order so that it occurs at a different part in the instance setup process, and absorbed it into the `install_python` function because it's a common first step for both EC2 and SM Local tests.

Reply:
Ack